include/runtime/stackmgr.h

```diff
@@ -74,6 +75,11 @@ class StackManager {
     return Span<Value>(ValueStack.end() - N, N);
   }
 
+  Span<Value> getRangeSpan(uint32_t Start, uint32_t End) {
```

Comment:
The name of `End` makes no sense. The constructor of `Span<T>` is `(BeginOffset, Length)`.

Reply:
noted